src/plenigo/models/OrderItem.php

```diff
@@ -0,0 +1,286 @@
+     */
+    public function getTaxes() {
+        return $this->taxes;
+    }
+
+    /**
+     * Getter method
+     * @return string
+     */
+    public function getTaxCountry() {
+        return $this->taxCountry;
+    }
+
+    /**
+     * Getter method
+     * @return int
+     */
+    public function getQuantity() {
+        return $this->quantity;
+    }
+
+    /**
+     * Getter method
+     * @return string
+     */
+    public function getCostCenter() {
+        return $this->costCenter;
+    }
+
+    /**
+     * Getter method
+     * @return string
+     */
+    public function getRevenueAccount() {
+        return $this->revenueAccount;
+    }
+
+    /**
+     * Getter method
+     * @return string
+     */
+    public function getStatus() {
+        return $this->status;
+    }
+
+    /**
+     * Setter method
+     * @param string $productId
+     */
+    public function setProductId($productId) {
+        $this->productId = $productId;
+    }
+
+    /**
+     * Setter method
+     * @param string $title
+     */
+    public function setTitle($title) {
+        $this->title = $title;
+    }
+
+    /**
+     * Setter method
+     * @param number $price
+     */
+    public function setPrice($price) {
+        $this->price = $price;
+    }
+
+    /**
+     * Setter method
+     * @param number $shippingCosts
+     */
+    public function setShippingCosts($shippingCosts) {
+        $this->shippingCosts = $shippingCosts;
+    }
+
+    /**
+     * Setter method
+     * @param number $taxes
+     */
+    public function setTaxes($taxes) {
+        $this->taxes = $taxes;
+    }
+
+    /**
+     * Setter method
+     * @param string $taxCountry
+     */
+    public function setTaxCountry($taxCountry) {
+        $this->taxCountry = $taxCountry;
+    }
+
+    /**
+     * Setter method
+     * @param int $quantity
+     */
+    public function setQuantity($quantity) {
+        $this->quantity = $quantity;
+    }
+
+    /**
+     * Setter method
+     * @param string $costCenter
+     */
+    public function setCostCenter($costCenter) {
+        $this->costCenter = $costCenter;
+    }
+
+    /**
+     * Setter method
+     * @param string $revenueAccount
+     */
+    public function setRevenueAccount($revenueAccount) {
+        $this->revenueAccount = $revenueAccount;
+    }
+
+    /**
+     * Setter method
+     * @param string $status
+     */
+    public function setStatus($status) {
+        $this->status = $status;
+    }
+
+    /**
+     * Creates an OrderItem array from an array of maps containing the fields
+     * 
+     * @param array $arrParam the array of maps containning the fields
+     * @return array an array of OrderItem objects 
+     */
+    public static function createFromArray($arrParam = array()) {
+        $res = array();
+
+        if (!is_null($arrParam) && is_array($arrParam) & count($arrParam) > 0) {
+            foreach ($arrParam as $oItem) {
+                $res[] = static::createFromMap($oItem);
```

Comment:
array.ṕush needs me to create a variable instead of just sending the result of a function. But I will evaluate other instances of array.push to see if I can save some variables.